### ISSUE TITLE
Content.GetInfo(contentId) always returns empty table

### ISSUE THREAD

**Issue by EmiMods** [XboxUnity/AuroraScripts#32]
Content.GetInfo(contentId) appears to always return an empty table (#table == 0).

To provide a bit of information on my usecase, I've been testing using contentIDs between 252 and 624. My cutoff of 624 appears to correlate to 252 + the number of visible titles in Aurora, which is 372 for me, but that may just be coincidence. Testing with Aurora 0.7b.2.

**Comment by Swizzy** [comment]
Do they exist in the database?

**Comment by EmiMods** [comment]
I believe they do exist in the database. Inside Content.db, the ContentItems table has entries with contentID of 252-382. When using Content.GetInfo(335), which is Black Ops in the database, I still get a table of 0 length.

Content is indeed defined in my script permissions. If I try to use Content.GetInfo on a number outside of 252-624 I get a nil response so the method does appear to get accessed.

**Comment by Swizzy** [comment]
It does indeed sound like a bug, and having a quick look at the code tells me what the problem likely is, i can't verify anything right now, but - we'll look into it closer and fix it.

This is one of those features that haven't been used by anyone yet i think :D

**Comment by EmiMods** [comment]
Yeah I wasn't able to find a single script using it so I think you're right. Thank you!

**Comment by Ste1io** [comment]
@EmiMods run this and let me know what it logs:

```lua
local function testContentInfoQueries()
    print("calling Content.FindContent()...")
    local found = Content.FindContent()
    print(#found .. " items.")

    if #found > 0 and found[1] then
        local findResult = found[1]
        print(string.format("[1] = {name='%s', id=%d, ..}", findResult.Name, findResult.Id))

        print("calling GetInfo(" .. findResult.Id .. ")...")
        local getResult = Content.GetInfo(findResult.Id)

        if getResult then print(getResult) end
    end
end
```

**Comment by EmiMods** [comment]
@Ste1io Issue was between chair and keyboard. 

TL;DR I was mapping the table length incorrectly. You put me on the right track and I sorted it out, thank you.
